ssstar/src/async_bridge.rs

```diff
@@ -19,7 +19,7 @@ use tokio::io::AsyncWrite;
 /// performed in a blocking worker thread, using [`tokio::task::spawn_blocking`].
 pub(crate) fn stream_as_reader<S>(stream: S) -> impl Read
 where
-    S: Stream<Item = Result<Bytes>> + Send + 'static,
+    S: Stream<Item = Result<Bytes>> + Send + Sync + 'static,
```

Comment:
I think for this minor change we need to update a patch version to 0.3.1, is it right?

Reply:
Yes, only patch needs to be updated.

Reply:
Yes, but only because this function is `pub(crate)`.  If it were part of the public API this would be a breaking change and require 0.4.0 release.

Reply:
@anelson Thanks, this is useful knowledge.